### DISCUSSION TITLE
Error: I cant open my project

### DISCUSSION THREAD

**Discussion by Marlonep** [AntlerVC/firetable#929] [Support / Q&A] [answered]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Rowy Run version**
If applicable, share the Rowy Run version from your project settings.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by harinij** [comment, marked as answer]
Hi @Marlonep if this is still an issue, it would be helpful if you can share some details with screen recording or email to support@rowy.io so we can help resolve this.
